site/pages/api/pledge/index.ts

```diff
@@ -8,16 +9,17 @@ export default async function handler(
   switch (req.method) {
     case "PUT":
       try {
-        const sessionToken = req.headers.authorization?.split(" ")[1];
-        const data = await findOrCreatePledge({
-          pledge: req.body,
-          sessionToken,
+        const signature = req.headers.authorization?.split(" ")[1] as string;
```

Comment:
ditch the assert, if there is no authorization header we should return `400` bad request

Reply:
good one!